ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -111,6 +111,7 @@ public struct SignalProducer<T, E: ErrorType> {
 		lock.name = "org.reactivecocoa.ReactiveCocoa.SignalProducer.buffer"
 
 		var events: [Event<T, E>] = []
+		var terminationEvent : Event<T,E>?
```

Comment:
Can you please remove the space to the left of the `:`, to match the surrounding code?